website/src/categories/components/switch/usage.md

```diff
@@ -98,32 +68,39 @@ If your setting has multiple states or requires more than a simple on/off toggle
 
 The switch provides a set of options, ensuring it's ready to adapt to your unique use cases.
 
-{{ 'components/switch/sl-switch-figma-options.svg' | svgImage }}
+<div class="ds-table-wrapper">
+  
+### Switch
 
 |Item|Options|Description|
 |-|-|-|
-|Sizes|`'small', 'medium', 'large'`|...|
-|Status |`...`|...|
-|State|`...`|...|
-|Icon|`...`|...|
+|Sizes|`sm` `md` `lg`|The switch come in three sizes: small, medium (default) and large. |
+|Orientation |`left` `right` `vertical`|The switch has 3 different lay-out possibilities, providing flexibility to align with your design requirements.|
```

Comment:
We no longer have the `vertical` orientation in the code, maybe it should be updated in Figma as well?

Reply:
will remove it here in the documentation and make an issue to remove it figma